library/spdm_crypt_lib/libspdm_crypt_crypt.c

```diff
@@ -12,8 +12,12 @@
 #endif
 
 /*leaf cert basic constraints len*/
-#ifndef BASIC_CONSTRAINTS_LEN
-#define BASIC_CONSTRAINTS_LEN 2
+#ifndef BASIC_CONSTRAINTS_CA_LEN
+#define BASIC_CONSTRAINTS_CA_LEN 2
```

Comment:
I think we need change this to 5, with comment below:
+--------------------------------------------+
| seq | seq_len | bool | bool_len | bool_val |
+--------------------------------------------+
| 30 |   03  | 01 |  01  | 00/ff |
+--------------------------------------------+

Reply:
please also add description on why optional length is not meaningful, with URL.